CITATION.cff

```diff
@@ -178,3 +178,16 @@ references:
     notes: >-
       MapReduce techniques are used via Parsl apps and workflow configuration
       to help achieve scalable data engineering for CytoTable.
+  - authors:
+      - name: "Colas Lab"
+    date-accessed: "2024-01-09"
+    title: Colas Lab Example IN Carta Dataset
+    type: data
+    notes: >-
+      Colas Lab provided access to dataset created from IN Carta for
```

Comment:
```suggestion
      The Colas Lab provided access to a single-cell dataset exported from IN Carta Image Analysis Software (Molecular Devices) for
```

This might violate character count, feel free to ignore this suggestion and add yourself if easier

Reply:
Thanks, this is great, I'll add in and adjust if the character count per line or other aspects are in violation of linting / etc.